invenio_curations/assets/semantic-ui/js/invenio_curations/deposit/DepositBox.js

```diff
@@ -147,12 +147,20 @@ export class DepositBoxComponent extends React.Component {
     }
   };
 
+  setGreenLightForCurationRequest = () => {
+    let positiveFeedback = document.getElementById("positive-feedback-div") != null;
+    let infoFeedback = document.getElementById("info-feedback-div") != null;
+    
+    return positiveFeedback || infoFeedback;
+  };
```

Comment:
does this mean, the validation check is based on the existence of a non-negative feedback box?
feels a bit like a workaround, though half the package consists of workarounds anyway :sweat_smile: